docker/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM openjdk:8-jre-alpine
+
+ENV APP_PORT 8086
+ENV WHEREABOUTS_VERSION "0.0.1-SNAPSHOT"
+# to be removed and replaced with WHEREABOUTS_VERSION after first release
+ENV WHEREABOUTS_SNAPSHOT_VERSION "0.0.1-20190606.073652-5"
+ENV WHEREABOUTS_JAR_URL https://repo.thehyve.nl/content/repositories/snapshots/nl/thehyve/whereabouts/${WHEREABOUTS_VERSION}/whereabouts-${WHEREABOUTS_SNAPSHOT_VERSION}.jar
```

Comment:
This also works: https://repo.thehyve.nl/service/local/artifact/maven/redirect?r=snapshots&g=nl.thehyve&a=whereabouts&v=0.0.1-SNAPSHOT&p=jar

Reply:
Will get you the latest snapshot.

Reply:
great! thanks, I was looking for this possibility, but couldn't find a proper pattern for the url

Reply:
actually this gives me: "307 Temporary Redirect"

Reply:
Weird, the `wget` in BusyBox apparently does not handle redirects.
This works:
```docker
RUN adduser -h "${APP_USR_HOME}" -D ${APP_USR} && \
    sed -i 's/\r//' /entrypoint.sh && \
    chown "${APP_USR}" /entrypoint.sh && \
    chmod u+x /entrypoint.sh && \
    # download the jar
    apk add curl && \
    curl -L "${WHEREABOUTS_JAR_URL}" -o "${APP_USR_HOME}/whereabouts-${WHEREABOUTS_VERSION}.jar"
```

Reply:
nice! it works indeed! I updated the PR.